src/main.cpp

```diff
@@ -17,6 +27,21 @@ int main() {
 
   uint32_t i = 0;
   while (1) {
+    char buf[128];  // buffer to hold bytes read
```

Comment:
Should we be using `char` or something more like uint16_t?

Reply:
This is specifically for console reads (nonblocking console). Pixels / camera integration aren't part of the skeleton code. Added comments to clarify.